core/server/common/src/test/java/alluxio/master/StateLockManagerTest.java

```diff
@@ -140,6 +139,7 @@ public void testExclusiveOnlyMode() throws Throwable {
   }
 
   @Test
+  // TODO(jiacheng): run this test before committing
```

Comment:
we should put in a static check for no TODO comments 😭

Reply:
oh shoot i forgot to remove this TODO... These UTs pass before I merged so we are okay per se in terms of quality.